files/find.py

```diff
@@ -313,7 +313,7 @@ def main():
                        continue
 
                     try:
-                        st = os.stat(fsname)
+                        st = os.lstat(fsname)
```

Comment:
I agree on expanding filetypes, this should also be a list, so you can select multiple file types.